vexim/siteaddsubmit.php

```diff
@@ -132,10 +132,6 @@
       } else {
         header ("Location: site.php?added={$_POST['domain']}" .
                 "&type={$_POST['type']}");
-        mail("{$_POST['localpart']}@{$_POST['domain']}",
```

Comment:
Oh, if this is indeed a relay domain, then you're right, there's no need to send the email.